qemu/tests/cfg/iperf_test.cfg

```diff
@@ -43,3 +43,13 @@
                 - host2guest:
                 - guest2host:
                     iperf_server = ${main_vm}
+        - with_short_packets:
+            queues = 1
```

Comment:
I guess we only need run this test case with virtio_net?If yes, please specify it.
`only virtio_net`